autoload/vista/extension/rst.vim

```diff
@@ -0,0 +1,89 @@
+" Heavily inspired  from https://raw.githubusercontent.com/Shougo/unite-outline/master/autoload/unite/sources/outline/defaults/rst.vim
```

Comment:
You can add the header for this new file :).

```vim
" Copyright (c) 2019 Your Name
" MIT License
" vim: ts=2 sw=2 sts=2 et
```

Reply:
done in 83fcdbe